daft/dataframe/dataframe.py

```diff
@@ -177,6 +199,7 @@ def explain(
             file (Optional[io.IOBase]): Location to print the output to, or defaults to None which defaults to the default location for
                 print (in Python, that should be sys.stdout)
         """
+        self.explain_broadcast()
```

Comment:
Is this correct? Why would be broadcast to the dashboard if we're not executing a query

Reply:
Hmm, I was thinking if a user just wanted to see what plan would be produced without executing a query, they could use the UI to also view the query plan.

In this case, the UI would show:
- the query plan
- the time it took to produce the plan
- "N/A" for the time it took to execute the query (since `.explain` doesn't execute queries)